src/playstate/preparenext.asm

```diff
@@ -44,11 +44,29 @@ playState_prepareNext:
         lda linecapState
         cmp #LINECAP_HALT
         bne @linecapHaltEnd
+		lda crashFlag
+		cmp #$F0
+		bne @gg
+		lda #'C'
```

Comment:
This might be a case for the version of [copyGraphicToPlayfield](https://github.com/kirjavascript/TetrisGYM/blob/6849035f26043c6ec2cc57cc4ca7d83a76b23905/src/playstate/preparenext.asm#L83) that's currently in #54